src/instruments/src/EFB/Ground/Pages/Payload/Payload.tsx

```diff
@@ -723,6 +751,8 @@ export const Payload = () => {
                                         </div>
                                     </TooltipWrapper>
 
+                                    <div className="flex relative flex-row items-center ml-4 text-sm font-light">{`Estimated duration: ${calculateBoardingTime()} minutes`}</div>
```

Comment:
I recommend putting this after "Loading Time".
Also make sure it works with translations. 
Use t() as in other examples on this page and add the translation key to en.json - post on Discord channel #localisation that you need this added to translation. 
Make sure it fits the layout also for languages with longer translations,